packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -106,32 +107,34 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
   }
 
   private _renderMultiLine(): React.ReactElement<React.HTMLAttributes<HTMLAreaElement>> {
-    const { theme } = this.props;
     return (
-      <div style={{ background: theme!.semanticColors.bodyBackground }}>
+      <div style={{ background: getTheme().semanticColors.bodyBackground }}>
```

Comment:
Do you know of a specific case where the theme prop was undefined?

Reply:
No this was done because using the theme prop passed in was making it use the variant theme instead of the global theme.